gomock/callset.go

```diff
@@ -87,8 +87,13 @@ func (cs callSet) FindMatch(receiver interface{}, method string, args []interfac
 		}
 	}
 
-	if len(expected)+len(exhausted) == 0 {
+	expectedNum := len(expected)
+	exhaustedNum := len(exhausted)
+
+	if expectedNum == 0 && exhaustedNum == 0 {
```

Comment:
Is this check safe to change? I don't think it is possible for len to ever be less than zero so it seems fine to me.